google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/options/GoogleComputeEngineTemplateOptions.java

```diff
@@ -73,6 +88,65 @@ public boolean autoCreateKeyPair() {
       return autoCreateKeyPair;
    }
 
+   public boolean canIpForward() {
+      return canIpForward;
+   }
+
+   public void canIpForward(boolean enable) {
+      this.canIpForward = enable;
+   }
+
+   /**
+    * @see #getDisks()
+    * @see org.jclouds.googlecomputeengine.domain.templates.InstanceTemplate.AttachDisk
+    */
+   public GoogleComputeEngineTemplateOptions addDisk(AttachDisk disk) {
+      this.disks.add(disk);
+      return this;
+   }
+
+   public void autoCreateDisk(final AutoCreateDiskOptions diskOptions) {
+      this.autoCreateDiskOptions = diskOptions;
+   }
+
+   public AutoCreateDiskOptions getAutoCreateDiskOptions() {
+      return autoCreateDiskOptions;
+   }
+
+   /**
+    * @return the AttachDisks for this instance.
+    */
+   public List<AttachDisk> getDisks() {
+      return disks;
+   }
+
+   // /**
+   // * @see #shouldKeepBootDisk()
+   // */
+   // public GoogleComputeEngineTemplateOptions keepBootDisk(boolean
```

Comment:
In general there isn't commented out code in the repo so these will have to go before check-in.